.github/ISSUE_TEMPLATE/e--keras-applications-users.md

```diff
@@ -0,0 +1,12 @@
+---
+name: e) `keras.applications` users
+about: Select this if your issue is related to the `keras.applications` module.
+
+---
+
+
+The [keras.applications](https://github.com/keras-team/keras-applications) module has been splitted from the main Keras repository.
+
+Please submit your issue using this [link](https://github.com/keras-team/keras-applications/issues/new/choose).
```

Comment:
it should be https://github.com/keras-team/keras-applications/issues/new because there is no setup of multiple issue types in keras_applications.